src/specs/api.v2.yaml

```diff
@@ -1039,6 +1039,60 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/HTTPError"
+  "/experiments/{experimentId}/cellLevel/{fileId}/{fileName}":
```

Comment:
why do we pass both file name and file id? Isn't just the id enough?

Reply:
Can we cann the endpoint `cellLevelMeta` -- for consistency

Reply:
No, the file Id is not saved in S3 so it needs it to determine the file name on download when generating a signedURL